UI/Navigation/src/Navigation/Navigation.csproj

```diff
@@ -1,13 +1,6 @@
 <Project Sdk="Uno.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>
-      net8.0-android;
-      net8.0-ios;
-      net8.0-maccatalyst;
-      net8.0-windows10.0.19041;
-      net8.0-desktop;
-      net8.0-browserwasm;
-    </TargetFrameworks>
+    <TargetFrameworks>net8.0-android;net8.0-ios;net8.0-maccatalyst;net8.0-windows10.0.26100;net8.0-browserwasm;net8.0-desktop</TargetFrameworks>
```

Comment:
Was it necessary to bump the net8.0-windows10 TFM? I see some build failures surrounding this on the CI

Reply:
😮 It was to get rid of the local "unsafe code" build errors.